CHANGELOG.md

```diff
@@ -8,10 +8,19 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Don't add main for app specs.  
+  [Derek Ostrander](https://github.com/dostrander)
+  [#8235](https://github.com/CocoaPods/CocoaPods/pull/8235)
+
 * Add documentation for the Podfile installation options  
   [Eric Amorde](https://github.com/amorde)
   [#8198](https://github.com/CocoaPods/CocoaPods/issues/8198)
   [guides.cocoapods.org #142](https://github.com/CocoaPods/guides.cocoapods.org/issues/142)
+  
+* Adds app spec project integration.  
```

Comment:
ha I think I already took care of that in another PR

Reply:
👍

Reply:
you do not need this change at all.